### PR TITLE
Acl

### PR DESCRIPTION
**Pull request by EmilyMcCarthy**

### CODE REVIEW

File: src/main/java/com/cloudmine/api/persistance/CloudMineObject.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import java.lang.annotation.RetentionPolicy;
+
+/**
+ * <br>
+ * Copyright CloudMine LLC. All rights reserved<br>
+ * See LICENSE file included with SDK for details.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+public @interface CloudMineObject {
+    public static final String DEFAULT_VALUE = "Unset";
```
Comment (marcweil) [inline src/main/java/com/cloudmine/api/persistance/CloudMineObject.java:13]: Do you think this is a little dangerous? Maybe something a little less likely to be accidentally be a real user value?

Comment (EmilyMcCarthy) [inline src/main/java/com/cloudmine/api/persistance/CloudMineObject.java:13]: CloudMineObject no longer exists so that value is not used

Comment (marcweil) [inline src/main/java/com/cloudmine/api/persistance/CloudMineObject.java:13]: o

File: src/main/java/com/cloudmine/api/rest/options/CMSharedDataOptions.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+import com.cloudmine.api.Immutable;
+import com.cloudmine.api.rest.BaseURL;
+
+/**
+ * <br>
+ * Copyright CloudMine LLC. All rights reserved<br>
+ * See LICENSE file included with SDK for details.
+ */
+public class CMSharedDataOptions implements BaseURL {
+
```
Comment (marcweil) [inline src/main/java/com/cloudmine/api/rest/options/CMSharedDataOptions.java:12]: Moar docs?

**Comment by marcweil** [comment]
All looks good for the first pass.

**Pull request closed without merging**